themes/tiddlywiki/vanilla/base.tid

```diff
@@ -2186,3 +2186,7 @@ body.tc-dirty span.tc-dirty-indicator, body.tc-dirty span.tc-dirty-indicator svg
 	background: #f00;
 	color: #fff;
 }
+
+.tc-keyboard {
+	display: inline;
```

Comment:
> was referring to it having to be a block-level element

Ah, I see. Other widgets (eg button and reveal) have established the convention of a `tag` attribute that allows the default generated HTML element to be overridden.